stl/inc/chrono

```diff
@@ -5138,53 +5138,76 @@ namespace chrono {
 
     template <class _CharT, class _Traits, class _Alloc, class _Parsable>
     struct _Time_parse_iomanip {
-        _Time_parse_iomanip(const basic_string<_CharT, _Traits, _Alloc>& _Fmt_, _Parsable& _Tp_,
+        _Time_parse_iomanip(const _CharT* _Fmt_, _Parsable& _Tp_,
             basic_string<_CharT, _Traits, _Alloc>* _Abbrev_ = nullptr, minutes* _Offset_ = nullptr)
             : _Fmt{_Fmt_}, _Tp{_Tp_}, _Abbrev{_Abbrev_}, _Offset{_Offset_} {}
+        _Time_parse_iomanip(_Time_parse_iomanip&&) = delete;
 
-        const basic_string<_CharT, _Traits, _Alloc>& _Fmt;
+        const _CharT* _Fmt;

```

Comment:
This changes the member type of `_Time_parse_iomanip`. If this is considered to break ABI, a new class template can be used for the `const charT*` cases.

Reply:
Yes, it looks like this is going to trigger ODR violations when linking old and new code.